src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DynamicListItem.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
```

Comment:
minor thing, in theory, Value is better because it's thread-safe. For future reference :)